recipes/dask-cuda/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "dask-cuda" %}
+{% set version = "0.18.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dask-cuda-{{ version }}.tar.gz
+  sha256: f5127667bbb6730e23e22016fb1a64f308d8e8455fa61993990e5d237df75396
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - dask-cuda-worker = dask_cuda.cli.dask_cuda_worker:go
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - dask >=2.9.0
+    - distributed >=2.18.0
+    - numba >=0.40.1
```

The requirements in `setup.py` are unfortunately out-of-date. More up-to-date information in [this recipe]( https://github.com/rapidsai/dask-cuda/blob/branch-0.20/conda/recipes/dask-cuda/meta.yaml ). We are working on fixing this here ( https://github.com/rapidsai/dask-cuda/pull/573 )

```suggestion
    - numba >=0.50.0,!=0.51.0
```

---

Done!